README.md

```diff
@@ -44,7 +44,7 @@ Application used to test the functionality of the various library APIs.
 
 ### Locally generating reference documentation
 
-If you would like to locally generate reference documentation for TeamsJS v2, simply utilize the script `yarn docs` either from the monorepo root or inside the teams-js project root (`packages/teams-js`). This should output the generated documentation to `packages/teams-js/docs`.
+If you would like to locally generate reference documentation for TeamsJS v2, simply utilize the script `pnpm run docs` either from the monorepo root or inside the teams-js project root (`packages/teams-js`). This should output the generated documentation to `packages/teams-js/docs`.
```

Comment:
Why do we have to use the `run` keyword for the `docs` script command?

Reply:
When `pnpm docs` is ran it just open a `https://www.npmjs.com/package/teams-js-monorepo` after debugging I found out that `typedoc` doesn't understand `pnpm` symlink to `typescript` therefore we need to use `pnpm run docs` instead, which helps `typedoc` to point to correct `typescript`.